lib/LaTeXML/Package/statements.sty.ltxml

```diff
@@ -159,7 +159,7 @@ sub definitionBody {
       $doc->openElement('dc:title');
       $doc->absorb($title);
       $doc->closeElement('dc:title');}
-    $doc->openElement('omdoc:CMP');
+   if ($doc->isCloseable('omdoc:CMP')) { $doc->openElement('omdoc:CMP'); }
    $doc->absorb($props{body}) if $props{body};
    $doc->maybeCloseElement('omdoc:CMP');
```

Comment:
Yes maybeCloseElement does exist. See [LaTeXML::Core:Document](http://dlmf.nist.gov/LaTeXML/manual/coremodules/LaTeXML_Core_Document.html). 

I think it is `maybeOpenElement` that he doesn't want to introduce.